credentials/src/main/kotlin/Validation.kt

```diff
@@ -1,62 +1,57 @@
 package web5.credentials
 
-import java.util.Date
+import java.util.*
```

Comment:
what's the thinking behind importing the kitchen sink vs explicit imports?

Reply:
Ah, it's the editorconfig. Fixed this now.